stable/grafana/templates/deployment.yaml

```diff
@@ -75,6 +75,10 @@ spec:
               value: "{{ .Values.sidecar.dashboards.label }}"
             - name: FOLDER
               value: "{{ .Values.sidecar.dashboards.folder }}"
+            {{- if .Values.sidecar.dashboards.searchNamespace }}
+            - name: NAMESPACE
+              value: "{{ .Values.sidecar.dashboards.searchNamespace }}"
```

Comment:
Maybe namespace is better.

Reply:
That's a fair point. I chose the name because I thought it was more obvious for someone coming to this for the first time - the person would be coming from the point of view of trying to configure datasources/dashboards - rather than an inherent knowledge of how the sidecar behaves.